iset.mm

```diff
@@ -123757,14 +123757,6 @@ seq m ( x. , F ) ~~> x ) ) \/
       AVMRVFVGVNRVFVAVOVMAUGQVDVHUHUIUJVFAVGULKZGHZVIVEVFASDVGSDVQVIUMVFAVMTZVF
       VGVNTAVGUNUOVFVPBGVFABVRVFBVLTUPUQURUSUT $.
 
-    $( The exponential function on the reals is nondecreasing.  (Contributed by
-       Mario Carneiro, 11-Mar-2014.)  (Revised by Jim Kingdon, 20-Dec-2022.) $)
-    efler $p |- ( ( A e. RR /\ B e. RR )
-        -> ( ( exp ` A ) <_ ( exp ` B ) -> A <_ B ) ) $=
-      ( cr wcel wa ce cfv clt wbr wn wi efltim ancoms con3d simpl reefcld simpr
-      cle lenltd lenlt 3imtr4d ) ACDZBCDZEZBFGZAFGZHIZJBAHIZJUFUERIABRIUDUHUGUC
-      UBUHUGKBALMNUDUFUEUDAUBUCOPUDBUBUCQPSABTUA $.
-
```

Comment:
Why is this proof so  much shorter than the proof of ~eflt? It had almost the same size as the new proof (for ~efle) which is based on ~eflt.
Maybe this proof should be kept as ALT version.

Reply:
`efle` is a biconditional, but `efler` is only one direction (hence the `r` in the name).

In both cases, `efltim` is the key to one direction (this pull doesn't change that). But this pull request proves the other direction, not via `ltord1` as set.mm does, but via continuity.